rust/nasl-c-lib/build.rs

```diff
@@ -0,0 +1,10 @@
+fn main() {
+    // Tell Cargo that if the given file changes, to rerun this build script.
+
+    cc::Build::new()
+        .file("c/nasl_aes_mac_gcm/nasl_aes_mac_gcm.c")
+        .compile("crypt");
+
+    println!("cargo:rerun-if-changed=src/nasl_aes_mac_gcm.c");
+    println!("cargo:rustc-link-lib=gcrypt");
```

Comment:
maybe it would make sense to statically link gcrypt? so that we don't have to install gcrypt on a client.